darwin/options.py

```diff
@@ -131,7 +131,7 @@ def __init__(self):
         )
         parser_convert.add_argument("format", type=str, help="Annotation import to convert to")
 
-        parser_convert.add_argument("output_dir", type=str, help="Where to store output files")
+        parser_convert.add_argument("-o", "--output_dir", type=str, help="Where to store output files")
```

Comment:
this is a rather major change as it would break existing usage the conversion in cli, let's discuss offline